evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -498,8 +504,8 @@ contract FluxAggregator is AggregatorInterface, Owned {
    */
   function startNewRound()
```

Comment:
While we're in here, can we change this to `requestRateUpdate()` to match the existing Aggregator?